wiki/feature/spectator.md

```diff
@@ -106,16 +101,12 @@ In cases where more specific control is required function `ace_spectator_fnc_upd
 
 ### 1.3 Spectatable Sides
 
-Spectatable sides can simply be considered an extra layer of filtering for the spectatable unit list. Again, there are two methods of controlling the spectatable sides:
+Spectatable sides can be considered another filter for the unit list to determine which units are available to spectate base on their side. As such, units can also be whitelisted from this filter as detailed above. There are two stages in the spectatable sides mechanism:
 
-- Side filter
 - Side list
+- Side filter
 
-The side list is exactly what it sounds like, a list of sides spectatable by the local client. However, unlike spectatable units the side list remains static and can only be updated manually. This is because the side filter is applied on top of the side list whenever the unit list is automatically maintained - meaning the unit list will update if the player changes side or if the side relations change.
-
-Note that the unit whitelist/blacklist also serves to override this side filtering mechanism.
-
-The default side list is `[west,east,resistance,civilian]` and to update it (on the local client) function `ace_spectator_fnc_updateSpectatableSides` can be used:
+The side list is a list of sides *possible* to spectate on the local client, by default this is always `[west,east,resistance,civilian]` and does not change. To update it (on the local client) function `ace_spectator_fnc_updateSpectatableSides` can be used:
```

Comment:
Added in latest commit, it still reads correctly without it, but I can see why it's not as fluid